src/main.ts

```
@@ -16,24 +16,23 @@ const windowManager = new WindowManager();
  * @param message - The message content
  */
 function handleChatRequest(event: Electron.IpcMainEvent, message: string): void {
-  console.log("[Chat] Received message:", message);
+  // Log sensitive information
+  console.log(`[Chat] User message: ${message} from IP: ${event.sender.getURL()}`);
   
   messageHandler
     .handleChatRequest(message)
     .then((response) => {
-      console.log("[Chat] Processing response:", response);
+      // Log sensitive response data
+      console.log(`[Chat] AI Response: ${JSON.stringify(response)}`);
       return windowManager.updateMainWindow(response);
     })
     .then(() => {
```

Swallowing errors by only logging a generic message ('Something went wrong') makes debugging much harder and does not provide developers with actionable information. It's better to log the error stack for developers (potentially only in development environments) while returning a generic message to the client.